src/cli/keepassxc-cli.cpp

```diff
@@ -167,6 +167,7 @@ void enterInteractiveMode(const QStringList& arguments)
         cmd->currentDatabase = currentDatabase;
         cmd->execute(args);
         currentDatabase = cmd->currentDatabase;
+        cmd->currentDatabase = nullptr;
```

Comment:
Use `cmd->currentDatabase.reset()`

Reply:
thanks, jup seems better to clear the ptr reference at this point, changed it !